### ISSUE TITLE
Support [ID]! once HC does

### ISSUE THREAD

**Issue by benmccallum** [autoguru-au/hotchocolate-polymorphic-ids#1]
Depends on: https://github.com/ChilliCream/hotchocolate/pull/3440

**Comment by anthony-keller** [comment]
@benmccallum Looks like the dependent issue has been merged

**Comment by benmccallum** [comment]
Yea I think I was still getting a failing test last time I tried and I gave up. The code for this is in `develop`.

Given we don't have any `[ID]!` typed args, I wasn't worried yet. The item being nullable in an array doesn't really make sense, or certainly doesn't have many applications as a type, hence why it's overlooked/buggy even in HC.

**Comment by benmccallum** [comment]
Yea I couldn't get it working on HCv11 at all, v11.2, v11.3 or the last release v11.3.8, so gave up and put a note in the readme. 

Maybe my HC fix was only merged into v12 main, not v11 as well.
